apps/financial-aid/web-veita/src/components/TableBody/TableBody.tsx

```diff
@@ -1,106 +1,103 @@
 import React from 'react'
-import { Text, Box, Button } from '@island.is/island-ui/core'
-import Link from 'next/link'
+import { Box, Text, Button } from '@island.is/island-ui/core'
 
 import * as tableStyles from '../../sharedStyles/Table.css'
 import cn from 'classnames'
-
-import {
-  Application,
-  getState,
-  getMonth,
-  ApplicationState,
-} from '@island.is/financial-aid/shared/lib'
-
-import {
-  GeneratedProfile,
-  GenerateName,
-} from '@island.is/financial-aid-web/veita/src/components'
-import {
-  calcDifferenceInDate,
-  getTagByState,
-} from '@island.is/financial-aid-web/veita/src/utils/formHelper'
+import { Colors } from '@island.is/island-ui/theme'
 
 interface PageProps {
-  application: Application
+  items: React.ReactNode[]
   index: number
-  onApplicationUpdate: (
-    applicationId: string,
-    state: ApplicationState,
-  ) => Promise<void>
+  identifier: string
+  onClick?: () => void
+  hasMaxWidth?: boolean
 }
 
-const TableBody = ({ application, index, onApplicationUpdate }: PageProps) => {
+type TextVariants =
```

Comment:
Are these not export from ui-core?

Reply:
Nope, they are not exported from the index file, so we cant import them 😢